### PR TITLE
lint + multiple vac fix

### PR DESCRIPTION
**Pull request by bogotros**
line length lint fix
remove url after loop is finished so all urls are checked

### CODE REVIEW

File: ext/vac.py [hunk 1]
```diff
@@ -104,15 +104,20 @@ def write_vac(self):
 
     async def check_vac_status_and_send_results(self, channel: TextChannel = None, send_if_no_results: bool = False):
         response = []
+        banned = []
         for url in self.urls:
             if await check_vac_status(url):
+                csgo_mention = '@&454124197048745996'
+                poggers = 'guild_tools.get_emoji_str('poggers')'
```
Comment (blerud) [inline ext/vac.py:111]: ??

File: ext/vac.py [hunk 2]
```diff
@@ -104,15 +104,20 @@ def write_vac(self):
 
     async def check_vac_status_and_send_results(self, channel: TextChannel = None, send_if_no_results: bool = False):
         response = []
+        banned = []
         for url in self.urls:
             if await check_vac_status(url):
+                csgo_mention = '@&454124197048745996'
```
Comment (blerud) [inline ext/vac.py:110]: might as well add this to config

File: ext/vac.py [hunk 3]
```diff
@@ -104,15 +104,20 @@ def write_vac(self):
...
+                poggers = 'guild_tools.get_emoji_str('poggers')'
                 response.append(
-                    "<@&454124197048745996> <{}> is VAC banned! {} Removing from checker.".format(
-                        url, guild_tools.get_emoji_str('poggers'))
+                    "<{}> <{}> is VAC banned! {} Removing from checker.".format(csgo_mention, url, poggers)
```
Comment (wjgan7) [inline ext/vac.py:113]: probably shouldn't @csgo for subsequent ones. maybe set csgo_mention to be blank for those.

Comment (blerud) [reply ext/vac.py:113]: lmao i think we're pinging the github user csgo

File: ext/vac.py (hunk 1 above)
Comment (bogotros) [reply ext/vac.py:111]: lmao fuk

**Comment by blerud** [comment]
> 
> 
> ran it through an online linter https://black.now.sh/ maybe itll finally stop complaining

we dont use the default black config so that doesnt work

File: ext/vac.py [hunk 4]
```diff
@@ -20,10 +20,10 @@ async def check_vac_status(url):
 class Vac(commands.Cog):
     def __init__(self):
         try:
-            with open(config.vac_file, 'r') as f:
+            with open(config.vac_file, "r") as f:
```
Comment (blerud) [inline ext/vac.py:23]: unchange the quotes

**Pull request merged**